assets/src/components/Treeview.js

```diff
@@ -132,6 +137,15 @@ export default class Treeview extends HTMLElement {
         return visibility;
     }
 
+    _dblclicked(item) {
+        if (item.type != 'group') {
```

Comment:
To avoid unwanted listeners, we could add `@dblclick` event listener only for groups here : https://github.com/3liz/lizmap-web-client/pull/4179/files#diff-25aa8d9133469826d98486d1fbe1d18f36ca5c72a569d73aa6e56c9c07d49836R65

Reply:
It might not be important so you can ignore this comment.